php-frontend/src/main/java/org/sonar/plugins/php/api/tree/expression/ArrayAccessTree.java

```diff
@@ -19,21 +19,31 @@
  */
 package org.sonar.plugins.php.api.tree.expression;
 
-import com.google.common.annotations.Beta;
+import javax.annotation.Nullable;
+
+import org.sonar.plugins.php.api.tree.Tree;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.plugins.php.api.tree.Tree'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)